service/src/main/java/uk/gov/hmcts/reform/fpl/enums/DocmosisTemplates.java

```diff
@@ -3,7 +3,8 @@
 public enum DocmosisTemplates {
     C6("FL-PLW-HNO-ENG-00092.docx", "Notice_of_proceedings_c6.pdf"),
     C6A("FL-PLW-HNO-ENG-00093.docx", "Notice_of_proceedings_c6a.pdf"),
-    SDO("FL-PLW-STD-ENG-00099.doc", "standard_directions_order.pdf");
+    SDO("FL-PLW-STD-ENG-00099.doc", "standard_directions_order.pdf"),
+    C21("FL-PLW-GOR-ENG-00218.docx","C21_Order");
```

Comment:
```suggestion
    C21("FL-PLW-GOR-ENG-00218.docx", "C21_Order");
```

Reply:
could also add the trailing underscore here 🤷‍♂

Reply:
hm good idea, I like that too if you're ok with it